apache/trafficcontrol
# Fix CacheStats api to allow upgrading

## What does this PR (Pull Request) do?


- [x] This PR fixes #5001 


## Which Traffic Control components are affected by this PR?


- Traffic Monitor
- Traffic Ops
- Traffic Stats (Still only calls `{tmFQDN}/publish/CacheStats`)

## What is the best way to verify this PR?
* For TO, ensure the following endpoints work when you have TO v4.1 + TM v5 and vice versa:
    * `cache/stats`
    * `cdns/capacity`
    * `cdns/health` and `cdns/{name}/health`
    * `deliveryservices/{id}/capacity`
* Ensure that TMs of each version can successfully communicate with one another.
* Verify TS is able to Poll both v4.1 and v5 TMs. (Previously v5 TMs would fail)


## If this is a bug fix, what versions of Traffic Control are affected?

- master (5153889)

## The following criteria are ALL met by this PR




## Additional Information
